frontend/express/public/javascripts/countly/vue/components/nav.js

```diff
@@ -102,19 +102,12 @@
                     return t.name === name;
                 });
 
-                /*
-                    So if we are changing the url location, we dont need to emit the input event.
-                    Because it will rerender the view with the updated tab info and value will be
-                    reset automatically.
-
-                    Only emit input event when no route provided
-                */
                 if (tab.length && tab[0].route) {
                     if (this.noHistory) {
                         Backbone.history.noHistory(tab[0].route);
                     }
                     else {
-                        window.location.hash = tab[0].route;
+                        app.navigate(tab[0].route, true);
```

Comment:
Are you sure that (`.hash=` and `app.navigate`) both things yield the exact result?

Reply:
actually here the idea is when the route is present we want the backbone routing to trigger, so passing true as second parameter gives us the desired result.